content/en/agent/guide/agent-fips-proxy.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Datadog FIPS Compliance
+kind: Guide
+disable_toc: false
+further_reading:
+- link: "agent/proxy"
+  tag: "Documentation"
+  text: "Agent Proxy Configuration"
+algolia:
+  rank: 80
+  tags: ["fips", "fips proxy", "compliance"]
+---
+
+{{< site-region region="us,us3,us5,eu,ap1" >}}
+<div class="alert alert-warning">The Datadog FIPS proxy is only available in the US1-FED region.</a></div>
+{{< /site-region >}}
+
+The Datadog FIPS proxy ensures that communication between the Datadog Agent and Datadog uses FIPS-compliant encryption.
+
+The Datadog FIPS proxy is a separately-distributed component that is deployed on the same host as the Datadog Agent. The proxy acts as an intermediary between the Agent and Datadog. The Agent communicates with the Datadog FIPS proxy, which encrypts payloads using a FIPS 140-2 validated cryptographic module and relays the payloads to Datadog.
+
+## Supported platforms and limitations
+
+The Datadog FIPS proxy's compliance is based on its use of the FIPS 140-2 validated [Cryptographic Module - Certificate #4024][1]. See the related [security policy][2] for information around validated operating environments and restrictions. It is ultimately the end user's responsibility to ensure operating environment compliance with the security policy and wider FIPS guidance.
+
+Supported platforms (64-bit x86 only):
+- RHEL >= 7
+- Debian >= 8
+- Ubuntu >= 14.04
+
+**Note**: The Datadog FIPS proxy does not support Cluster Agent and Serverless Monitoring.
+
+## Prerequisites
+
+- TCP port range: 9803 to 9814
+- Datadog Agent >= v7.41
+
+## Install the Agent with FIPS support
+
+{{< tabs >}}
+{{% tab "Bare metal / VM" %}}
+To install the Datadog Agent with the FIPS proxy, add `DD_FIPS_MODE=1` to the one-step install instructions on the [Datadog Agent Integration][1] page. For example:
+
+```shell
+DD_API_KEY=<DD_API_KEY> \
+DD_SITE="ddog-gov.com" \
+DD_FIPS_MODE=1 \
+bash -c "$(curl -L \
+   https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
+```
+
+Setting the `DD_FIPS_MODE` environment variable installs the FIPS package along with the Agent, and configures the Agent to use the proxy. There are no additional configuration steps if you're using this method, but you should [verify the installation](#verify-the-installation).
+
+[1]: https://app.datadoghq.com/account/settings#agent/
+{{% /tab %}}
+
+{{% tab "Amazon ECS" %}}
+
+For instructions on installing the FIPS proxy on Amazon ECS, see the [FIPS proxy for GOVCLOUD environments][1].
+
+[1]:/containers/amazon_ecs/#fips-proxy-for-govcloud-environments
+{{% /tab %}}
+
+{{% tab "Helm on Amazon EKS" %}}
```

Comment:
This section (EKS/Helm) should be second and ECS as third given the likely prevalence of environments in the wild